logger.js

```diff
@@ -64,6 +64,18 @@ function pinoLogger (opts, stream) {
   }
 }
 
+function wrapReqSerializer (serializer) {
+  if (serializer === asReqValue) return asReqValue
+  return function wrappedReqSerializer (req) {
+    var _req = asReqValue(req)
+    Object.defineProperty(_req, 'raw', {
+      enumerable: false,
+      value: req
+    })
```

Comment:
Adding a new property via `defineProperty` in a serializer (which is a hot-path) is definitely __not a good idea__ perf-wise. We should transform `asReqValue` in a class, and attach this property to its prototype, and always memorize it internally through a `Symbol`.

Reply:
I have updated the PR. I hope I have addressed your concern correctly.